### PR TITLE
Make footer text configurable

### PR DESCRIPTION
**Pull request by toddrob99**
Use string values from settings.json instead of booleans with hard-coded footer text. Made strings null in sample_settings.json since the default value was false before. Added info to README.md so users will know how to format the footer text if they choose to enable it.

Also fixed missing newline after the Preview link in the game thread header when the rest of the header text is not yet available. Footer text was on the same line, now will start on the next line.

### CODE REVIEW

File: src/editor.py [hunk 1]
```diff
@@ -150,14 +150,14 @@ def generate_code(self,dir,thread):
             if self.line_score: code = code + self.generate_linescore(files)
             if self.scoring_plays: code = code + self.generate_scoring_plays(files)
             if self.highlights: code = code + self.generate_highlights(files)
-            if self.footer: code = code + self.generate_footer()
+            if self.footer: code = code + self.footer
```
Comment (mattabullock) [inline src/editor.py]: Instead of having the user include the `\n\n` could you just add them here and at the other spot?

File: src/editor.py [hunk 2]
```diff
@@ -194,7 +194,7 @@ def generate_header(self,files):
             timezone = self.time_zone
             date_object = date_object - t
             header = "**First Pitch:** " + date_object.strftime("%I:%M %p ") + timezone + "\n\n"
-            header = header + "[Preview](http://mlb.mlb.com/mlb/gameday/index.jsp?gid=" + game.get('gameday_link') + ")\n"
+            header = header + "[Preview](http://mlb.mlb.com/mlb/gameday/index.jsp?gid=" + game.get('gameday_link') + ")\n\n"
```
Comment (mattabullock) [inline src/editor.py:197]: This is unrelated to this PR, could you revert the change (or explain it)?

**Review by mattabullock — CHANGES_REQUESTED** [review]

File: src/editor.py (hunk 2 above)
Comment (toddrob99) [reply src/editor.py:197]: If the GDT is posted a few hours before game time, it is posted with only the Preview link and now the footer. Without this change, the footer text shows on the same line as the Preview link when the rest of the header text is not yet available.

File: src/editor.py (hunk 1 above)
Comment (toddrob99) [reply src/editor.py]: Done

**Review by toddrob99 — COMMENTED** [review]
Requested changes made/explained.

**Review by mattabullock — APPROVED** [review]
